src/code/z_skelanime.c

```diff
@@ -8,8 +8,9 @@ s32 func_801370B0(SkelAnime* skelAnime);
 s32 func_8013713C(SkelAnime* skelAnime);
 void SkelAnime_ChangeAnimDefaultRepeat(SkelAnime* skelAnime, AnimationHeader* animationSeg);
 void func_80137748(SkelAnime* skelAnime, Vec3f* pos, s16 angle);
-void SkelAnime_ChangeLinkAnim(GlobalContext* globalCtx, SkelAnime* skelAnime, LinkAnimetionEntry* linkAnimetionEntrySeg,
-                              f32 playbackSpeed, f32 frame, f32 frameCount, u8 animationMode, f32 transitionRate);
+void SkelAnime_ChangeLinkAnim(GlobalContext* globalCtx, SkelAnime* skelAnime,
+                              LinkAnimationHeader* LinkAnimationHeaderSeg, f32 playbackSpeed, f32 frame, f32 frameCount,
```

Comment:
Can the `LinkAnimationHeaderSeg` be called `linkAnimationHeaderSeg` everywhere instead?